arquillian-webcam-video-recorder/src/main/java/org/arquillian/extension/recorder/video/webcam/configuration/WebcamVideoRecorderConfigurator.java

```diff
@@ -0,0 +1,127 @@
+/*
+ * JBoss, Home of Professional Open Source
+ * Copyright 2013, Red Hat Middleware LLC, and individual contributors
+ * by the @authors tag. See the copyright.txt in the distribution for a
+ * full listing of individual contributors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.arquillian.extension.recorder.video.webcam.configuration;
+
+import org.arquillian.extension.recorder.video.*;
```

Comment:
I am not fan of these kinds of imports, I would be nice to see what is imported concretely. Just a hint to consider.